lib/lita/commands/notes.rb

```diff
@@ -0,0 +1,15 @@
+module Commands
+  class Notes
+    include Base
+
+    def call
+      incident_id = message.match_data['incident_id']
+      notes = pagerduty.get_notes_by_incident_id(incident_id)
+      response format_notes(notes, incident_id)
+    rescue Exceptions::IncidentNotFound
+      response message: 'incident.not_found', params: { id: incident_id }
```

Comment:
```suggestion
      response message: 'incident.not_found',
               params: { id: incident_id }
```

Reply:
Any profit of this?

Reply:
No profit.
Just saw it your code different styles one/multi-lines for `response`

It does not affect anything

Reply:
it depends on line length :)